pyproject.toml

```diff
@@ -33,11 +33,13 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.9"
 linkml-runtime = "^1.6.3"
+rdflib = "^7.0.0"
```

Comment:
oof if we are gaining rdflib as a runtime dependency just to use the `Namespace` class to declare a module constant i woudl call that highly undesirable and worthy of thinking about whether we need that